entries/t/trustage.com.json

```
@@ -0,0 +1,12 @@
+{
+    "Trustage": {
```

```suggestion
    "TruStage": {
```

Their name is TruStage